<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>propViewer</class>
 <widget class="QWidget" name="propViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>228</width>
    <height>328</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBoxView">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>View type:</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <property name="spacing">
          <number>4</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QRadioButton" name="radioPersp">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Perspective</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioOrtho">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Orthogonal</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QWidget" name="widget_2" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Angle of view:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinAngleOfView">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="suffix">
               <string>째</string>
              </property>
              <property name="prefix">
               <string/>
              </property>
              <property name="minimum">
               <number>15</number>
              </property>
              <property name="maximum">
               <number>170</number>
              </property>
              <property name="value">
               <number>35</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_3" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>View size (ortho):</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="spinOrthoViewSize">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimum">
               <double>0.000000000000000</double>
              </property>
              <property name="maximum">
               <double>99999999999999991433150857216.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="inputMethodHints">
           <set>Qt::ImhMultiLine</set>
          </property>
          <property name="text">
           <string>Brightness of background:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="spinBG">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="decimals">
           <number>2</number>
          </property>
          <property name="maximum">
           <double>1.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="value">
           <double>0.300000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Rotation</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="spinViewRotX">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="suffix">
              <string>째</string>
             </property>
             <property name="decimals">
              <number>6</number>
             </property>
             <property name="minimum">
              <double>-99999999.000000000000000</double>
             </property>
             <property name="maximum">
              <double>99999999.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>1.000000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="spinViewRotY">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="suffix">
              <string>째</string>
             </property>
             <property name="decimals">
              <number>6</number>
             </property>
             <property name="minimum">
              <double>-99999999.000000000000000</double>
             </property>
             <property name="maximum">
              <double>99999999.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>1.000000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="spinViewRotZ">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="suffix">
              <string>째</string>
             </property>
             <property name="decimals">
              <number>6</number>
             </property>
             <property name="minimum">
              <double>-99999999.000000000000000</double>
             </property>
             <property name="maximum">
              <double>99999999.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>1.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Translation</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="spinViewTransX">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="decimals">
              <number>6</number>
             </property>
             <property name="minimum">
              <double>-999999999.000000000000000</double>
             </property>
             <property name="maximum">
              <double>999999999.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.100000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="spinViewTransY">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="decimals">
              <number>6</number>
             </property>
             <property name="minimum">
              <double>-999999999.000000000000000</double>
             </property>
             <property name="maximum">
              <double>999999999.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.100000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="spinViewTransZ">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="decimals">
              <number>6</number>
             </property>
             <property name="minimum">
              <double>-999999999.000000000000000</double>
             </property>
             <property name="maximum">
              <double>999999999.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.100000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="horizontalWidget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Scale</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>x</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="spinViewScale">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="decimals">
            <number>6</number>
           </property>
           <property name="maximum">
            <double>360.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>spinViewRotX</sender>
   <signal>valueChanged(double)</signal>
   <receiver>propViewer</receiver>
   <slot>changedRotX(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>52</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>95</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinViewRotY</sender>
   <signal>valueChanged(double)</signal>
   <receiver>propViewer</receiver>
   <slot>changedRotY(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>52</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>95</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinViewRotZ</sender>
   <signal>valueChanged(double)</signal>
   <receiver>propViewer</receiver>
   <slot>changedRotZ(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>53</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>95</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinViewTransX</sender>
   <signal>valueChanged(double)</signal>
   <receiver>propViewer</receiver>
   <slot>changedTraXYZ(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>141</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>95</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinViewTransY</sender>
   <signal>valueChanged(double)</signal>
   <receiver>propViewer</receiver>
   <slot>changedTraXYZ(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>141</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>95</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinViewTransZ</sender>
   <signal>valueChanged(double)</signal>
   <receiver>propViewer</receiver>
   <slot>changedTraXYZ(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>141</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>95</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinAngleOfView</sender>
   <signal>valueChanged(int)</signal>
   <receiver>propViewer</receiver>
   <slot>changedAngle(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>138</x>
     <y>232</y>
    </hint>
    <hint type="destinationlabel">
     <x>95</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBG</sender>
   <signal>valueChanged(double)</signal>
   <receiver>propViewer</receiver>
   <slot>changedBGcolor(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>130</x>
     <y>202</y>
    </hint>
    <hint type="destinationlabel">
     <x>95</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinViewRotX</sender>
   <signal>editingFinished()</signal>
   <receiver>propViewer</receiver>
   <slot>rotXeditingFinished()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinOrthoViewSize</sender>
   <signal>valueChanged(double)</signal>
   <receiver>propViewer</receiver>
   <slot>changedOrthoViewSize(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>141</x>
     <y>243</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioPersp</sender>
   <signal>toggled(bool)</signal>
   <receiver>propViewer</receiver>
   <slot>radioPropToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>54</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>163</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>changedRotX(double)</slot>
  <slot>changedRotY(double)</slot>
  <slot>changedRotZ(double)</slot>
  <slot>changedTraXYZ(double)</slot>
  <slot>changedScale(double)</slot>
  <slot>changedBGcolor(double)</slot>
  <slot>changedAngle(int)</slot>
  <slot>rotXeditingFinished()</slot>
  <slot>changedOrthoViewSize(double)</slot>
  <slot>radioPropToggled(bool)</slot>
 </slots>
</ui>
